### PR TITLE
add ui for insights enablement

### PR DESCRIPTION
**Pull request by jlmitch5**
This adds support for the UI to set the new insights data enablement feature.

### CODE REVIEW

**Comment by ryanpetrello** [comment]
Can we combine this w/ @rooftopcellist's PR?

**Review by mabashian — APPROVED** [review]
lgtm

**Comment by unlikelyzero** [comment]
Can you include the things that were tested on this one? I think it's just toggling the feature on and off and verifying that it takes.

@rooftopcellist how is this configuration applied in a cluster topology?

File: awx/ui/client/src/license/license.partial.html [hunk 1]
```diff
@@ -115,25 +115,38 @@
...
+									<input type="checkbox" ng-model="newLicense.pendo" ng-disabled="!user_is_superuser" required>
+									<translate>Pendo</translate>
+							</div>
+							<div class="License-analyticsCheckbox checkbox">
+									<input type="checkbox" ng-model="newLicense.insights" ng-disabled="!user_is_superuser" required>
+									<translate>Insights</translate>
+							</div>
 						</div>
+						<span class="License-helperText">
+							<translate>For more information about track and analytics, see
```
Comment (rooftopcellist) [inline awx/ui/client/src/license/license.partial.html:143]: Lets change this to `For more information about Automation Insights and analytics, see...`

**Comment by unlikelyzero** [comment]
Verified:

- `Revert all to default` disables insights from the System Settings Page
- Insights can be disabled on install

**Comment by ryanpetrello** [comment]
@unlikelyzero have we verified the default behavior we want on install/license application (verified that people can opt out on the license UI)?

**Comment by unlikelyzero** [comment]
Yes, that's covered by the second bullet point

**Comment by ryanpetrello** [comment]
@jlmitch5 if your intention is to merge this into @rooftopcellist's branch, you need to push the `Merge pull request` button by hand.  The `mergeit` label is only for merges into to `devel` by zuul

**Comment by jlmitch5** [comment]
thanks @ryanpetrello  doing now

**Pull request merged**